demisto_sdk/commands/postman_codegen/tests/test_files/VirusTotal.postman_collection.json

```diff
@@ -28,8 +28,7 @@
 							"value": "https://www.cobaltstrike.com/"
 						}
 					]
-				},
-				"description": "URL Report description"
+				}
```

Comment:
@PA-Rweins I removed the description in this one, to make sure that the description is generated with empty string in the unit test.